recipes/scipp/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set version = "25.01.0" %}
+
+package:
+  name: scipp
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/scipp/scipp/archive/{{ version }}.tar.gz
+  sha256: ce410dc7f92e93785a7f8309c509fa09732e047fc8100069a97d577638f60d40
+
+build:
+  skip: true  # [py<311]
+  script_env:
+    - GIT_VERSION_INFO={{ version }}
+  number: 0
+  missing_dso_whitelist:
+    - '*'        # [win]
+  script:
+    - cmake --preset package-{{ target_platform }} -DPython_EXECUTABLE="%PYTHON%" -DCONDA_FORGE_BUILD=ON  # [win]
+    - cmake --preset package-{{ target_platform }} -DPython_EXECUTABLE=$PYTHON -DCONDA_FORGE_BUILD=ON  # [unix]
+    - cmake --build --preset build
+    - ctest --preset test  # [build_platform == target_platform]
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - python  # [build_platform != target_platform]
+    - cmake
+    - conan ==1.64.1
+    - ninja
+    - make  # [linux]
+  host:
+    - python
+    - tbb-devel ==2022.0.0
```

Comment:
Why are you pinning this to strictly? This will make it incompatible with most other packages in conda-forge, you should rather use the global pins for these dependencies.

Reply:
Thanks for the review! I have removed the pins where I can use the global pins instead.
For gtest/gmock, scipp's test suite does complain with the newer versions.

The linter failure seems unrelated.